storage/ndb/rest-server/rest-api-server/internal/integrationtests/ping/wrapper_test.go

```diff
@@ -52,5 +53,8 @@ func TestMain(m *testing.M) {
 	}
 	defer cleanup()
 
+	// Less than NumCPUs seemed to work well here:
+	runtime.GOMAXPROCS(conf.Internal.GOMAXPROCS)
```

Comment:
set the conf.Internal.GOMAXPROCS before calling integrationtests.InitialiseTesting

Reply:
Yeah it's actually entirely unnecessary here.